ImageLounge/src/DkCore/DkActionManager.cpp

```diff
@@ -1279,22 +1301,23 @@ void DkActionManager::createActions(QWidget* parent) {
 	mViewActions[menu_view_frameless]->setCheckable(true);
 	mViewActions[menu_view_frameless]->setChecked(false);
 
-	mViewActions[menu_view_new_tab] = new QAction(QObject::tr("New &Tab"), parent);
-	mViewActions[menu_view_new_tab]->setShortcut(QKeySequence(shortcut_new_tab));
-	mViewActions[menu_view_new_tab]->setStatusTip(QObject::tr("Open a new tab"));
-
-	mViewActions[menu_view_close_tab] = new QAction(QObject::tr("&Close Tab"), parent);
-	mViewActions[menu_view_close_tab]->setShortcut(QKeySequence(shortcut_close_tab));
-	mViewActions[menu_view_close_tab]->setStatusTip(QObject::tr("Close current tab"));
+	mViewActions[menu_view_first_tab] = new QAction(QObject::tr("F&irst Tab"), parent);
+	mViewActions[menu_view_first_tab]->setStatusTip(QObject::tr("Switch to first tab"));
 
 	mViewActions[menu_view_previous_tab] = new QAction(QObject::tr("&Previous Tab"), parent);
 	mViewActions[menu_view_previous_tab]->setShortcut(QKeySequence(shortcut_previous_tab));
 	mViewActions[menu_view_previous_tab]->setStatusTip(QObject::tr("Switch to previous tab"));
 
+	mViewActions[menu_view_goto_tab] = new QAction(QObject::tr("&Go to Tab"), parent);
```

Comment:
`Go to`, `First Tab`, and `Last Tab` are from my point of view too much - maybe we could add an option _extended tabs_  that adds these items to the menu (including open tabs & save tabs)?